### ISSUE TITLE
Add Julia Language section to VS Code docs

### ISSUE THREAD

**Issue by logankilpatrick** [julia-vscode/julia-vscode#2389]
Rust currently has a PR open to add docs specific for the language on: https://code.visualstudio.com/docs/languages/overview. We should also do the same!

**Comment by logankilpatrick** [comment]
https://github.com/microsoft/vscode-docs/blob/main/docs/languages/overview.md

**Comment by davidanthoff** [comment]
Yes, that would be fantastic! It might make sense to check with the VS Code team whether they would merge such a PR, the languages that are featured in the docs at the moment all are well above a million downloads for their respective extension, which we are pretty far away from... On the other, for the data science stuff we might be an attractive addition? @claudiaregio, do you have a sense whether that would be accepted?

**Comment by logankilpatrick** [comment]
They seemed willing to help with the open Rust PR but I don't know how many downloads the rust extension has.

**Comment by davidanthoff** [comment]
> I don't know how many downloads the rust extension has.

1.2M, we're at 240K. I have no idea whether that is the criterion, though.

**Comment by logankilpatrick** [comment]
Makes sense, happy to help with this if they are willing to accept it.

**Comment by davidanthoff** [comment]
I think it would be a _major_ plus if we managed to get that in, so definitely worth pursuing!
